ts/industries.ts

```diff
@@ -62,7 +62,7 @@ export const IndustryNames = [
     'MeatPackingPlant',
     'oilfield',
     'Refinery',
-    'Sandhouse',
+    'SandHouse',
```

Comment:
It looks like you based your code on an older revision. I recommend using git to create commits locally, so you don't keep running in to this issue.

Reply:
I changed that my self cause i did cross check the sand house from rrox dev plugin and commits locally